tests/testthat/test-check.r

```diff
@@ -29,3 +29,16 @@ test_that("aspell environment variables", {
     expect_warning(aspell_env_var(), "Skipping spell check")
   )
 })
+
+test_that("setting env_vars overrides defaults", {
```

Comment:
I'd rather you test this by pulling `rcmd_check_envars()` out into a separate function.